test/logic_value_test.dart

```diff
@@ -1022,4 +1022,201 @@ void main() {
           equals(LogicValue.filled(0, LogicValue.zero).hashCode));
     });
   });
+
+  group('Comparable LogicValue', () {
+    test('positive - int', () {
+      final a = LogicValue.ofInt(3, 8);
+      final b = LogicValue.ofInt(0, 8);
+      final c = LogicValue.ofInt(1, 8);
+      final d = LogicValue.ofInt(2, 8);
+      final e = LogicValue.ofInt(23, 8);
+      final f = LogicValue.ofInt(44, 8);
+      final g = LogicValue.ofInt(2, 8);
+      final h = LogicValue.ofInt(9, 8);
+
+      final values = <LogicValue>[a, b, c, d, e, f, g, h];
+
+      final expected = List<int>.filled(values.length, 0);
+
+      for (var i = 0; i < values.length; i++) {
+        expected[i] = values[i].toInt();
+      }
+
+      values.sort();
+      expected.sort();
+      for (var i = 0; i < values.length; i++) {
+        expect(values[i].toInt(), expected[i]);
+      }
+    });
+    test('signed values - int 64 bits', () {
```

Comment:
good test, but wrong result.  the `LogicValue`s should always compare as unsigned numbers.